src/Lhm/Lhm.php

```diff
@@ -152,12 +154,12 @@ public static function cleanup($run = false, array $options = [])
                 $adapter->query("DROP TRIGGER IF EXISTS {$trigger}");
             }
 
+            $adapter->query('SET foreign_key_checks=0');
```

Comment:
I took a quick look at Soundcloud/lhm and it does not seem to be disabling foreign keys. Would you be able to provide a test case that requires FK checks?